node/batcher/requests_inspector_verifier.go

```diff
@@ -46,7 +46,7 @@ type RequestsInspectorVerifier struct {
 	shards                   []types.ShardID
 	shardID                  types.ShardID
 	logger                   types.Logger
-	mapper                   *core.Router // TODO make this into an interface?
+	mapper                   *router.Mapper // TODO make this into an interface?
```

Comment:
can you also get this todo done? 
this is good for testing 
and if we ever want to replace the mapping function